simd.dd

```diff
@@ -60,6 +60,9 @@ import core.simd;
 	the compiler is targetting. If a particular CPU family has varying
 	support for vector types, an additional runtime check may be necessary.
 	The compiler does not emit runtime checks; those must be done by the programmer.
+    For instance, the Windows version of DMD only implements support for SIMD when
+    compiling with the 64bit flag. If a program does not contain runtime checks,
```

Comment:
I had decided to add it there because that documentation is the most
detailed. The other ones are more terse. Nevertheless, I would recommend
adding some kind of information about this to all three of those locations.
I hadn't thought of it until you mentioned it.

I have a Github client on my home computer. I doubt it will work on my work
computer (firewall).

On Wed, Jun 24, 2015 at 4:34 PM, Vladimir Panteleev <
notifications@github.com> wrote:

> In simd.dd
> <https://github.com/D-Programming-Language/dlang.org/pull/1032#discussion_r33192579>
> :
>
> > @@ -60,6 +60,9 @@ import core.simd;
> >  	the compiler is targetting. If a particular CPU family has varying
> >  	support for vector types, an additional runtime check may be necessary.
> >  	The compiler does not emit runtime checks; those must be done by the programmer.
> > +    For instance, the Windows version of DMD only implements support for SIMD when
> > +    compiling with the 64bit flag. If a program does not contain runtime checks,
>
> Maybe such a change belongs in the documentation for the core.simd module
> instead, or to the documentation of the D_SIMD built-in version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/D-Programming-Language/dlang.org/pull/1032/files#r33192579>
> .
>